pom.xml

```diff
@@ -88,23 +88,23 @@
 
         <!-- Dependencies -->
         <airline.version>0.7</airline.version>
-        <amqp-client.version>3.6.5</amqp-client.version>
+        <amqp-client.version>4.0.0</amqp-client.version>
```

Comment:
I haven't tested it, but according to https://github.com/rabbitmq/rabbitmq-java-client#4x-series, it should be compatible:

> Starting with 4.0, this client releases are independent from RabbitMQ server releases. These versions can still be used with RabbitMQ server 3.x.